linkatos/message.py

```diff
@@ -28,3 +28,13 @@ def is_list_request(message):
     list_found = list_re.search(message)
 
     return list_found is not None
+
+
+def purge_request(message):
+    purge_re = re.compile("(purge) (\d+)")
```

Comment:
I don' t understand how I can do this without compiling it.

And I'm not that happy with taking it out of the function, as I feel the regex belongs to this specific function.